bitrise.yml

```diff
@@ -57,6 +57,7 @@ workflows:
       - wait-for-android-emulator@1:
           inputs:
             - boot_timeout: 600
+      - restore-gradle-cache@1: { }
```

Comment:
Can we add this to `prepare_all` instead?

Reply:
sure! for this to work it has to be used in combination with bitrise steps that make use of the gradle cache keys (such as `android-build`)